documentation/existing.md

```diff
@@ -1,163 +1,176 @@
-  <td>Datacenter,<br> Datastore Cluster,<br> Datastore Storage Folder</td>
-  <td>Yes</td>
-</tr>
-</tbody>
-</table>
+[VirtualCenter "1.1.1.3"]
+[VirtualCenter "1.1.1.4"]
 
-Please note that the user configured for vSphere Cloud Provider will be used by all Kubernetes users on the same cluster. If one single user is not sufficient for different use cases, it is recommended to utilize [Kubernetes RBAC Authorization](https://kubernetes.io/docs/admin/authorization/rbac/) to drive authorization decisions, allowing admins to dynamically configure access policies through the Kubernetes API.
+[Workspace]
+server = "1.1.1.1"
+datacenter = "us-east"
+default-datastore="sharedVmfs-0"
+resourcepool-path=" "
+folder = "kubernetes"
 
-In some cases Administrator may still want to customize the privileges for vSphere Cloud Provider user:
+[Disk]
+scsicontrollertype = pvscsi
 
-1. SPBM support in vSphere Cloud Provider was introduced in Kubernetes 1.7 release. For 1.6 or older release, remove SPBM related privileges to keep required privileges to a minimal.
-2. Kubernetes RBAC mode was stabilized as of 1.8 release. For 1.7 or older release, Administrator may still want to customize the privileges for vSphere Cloud Provider user based on different use cases. It's recommended to assign different roles to the same vCenter user instead of switching to a different vCenter user, in which case Kubelet needs to be restarted, which might be a concern in production environment.
+[Network]
+public-network = "VM Network"
+```
 
+Below is the summary of supported parameters in the `vsphere.conf` file for Kubernetes version 1.9.x
 
-Here are [some examples](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/vcp-roles.html) to customize roles and privileges for vSphere Cloud Provider user.
+* Properties in ```Global``` section will be used for all specified vCenters unless overridden in ```VirtualCenter``` section.
+* ```port``` is the vCenter Server Port. The default is 443 if not specified.
+* ```insecure-flag``` should be set to 1 if the vCenter uses a self-signed cert.
+* ```datacenters``` should be the list of all comma separated datacenters where Kubernetes node VMs are present.
+* ```default-datastore``` is the default datastore to use for provisioning volumes using storage classes/dynamic provisioning.
+* ```Workspace``` section specify properties which will be used for various vSphere Cloud Provider functionality. e.g. Dynamic provisioning, Storage Profile Based Volume provisioning etc.
+   * ```server``` specified in this section should have the master node VM.
+   * ```datacenter``` is the name of datacenter on which specified VM folder, datastore and resource pool should be present.
+   * ```folder``` is the vCenter VM folder path in which dummy VMs should be created.
+   * ```resourcepool-path``` is the path to resource pool where dummy VMs for Storage Profile Based volume provisioning should be created. It is an optional parameter.
```

Comment:
is the path to resource pool where dummy VMs will be created. 

Another comment, a lot of these properties are deprecated and/or is optional. Can you also highlight those?